include/revng/EarlyFunctionAnalysis/FunctionSummaryOracle.h

```diff
@@ -140,7 +140,12 @@ class FunctionSummaryOracle {
     return LocalFunctions.at(PC);
   }
 
-  /// \return a description of the call and boolean indicating whether the call
+  /// \param Function caller function address
+  /// \param CallerBlockAddress caller basic block address
+  /// \param CalledLocalFunction callee function address
+  /// \param CalledSymbol name of dynamic symbol we are calling (can be
+  /// extracted from llvm::Value with extractFromConstantStringPtr) \return a
+  /// description of the call and boolean indicating whether the call
```

Comment:
Mind the indentation
```suggestion
  /// \param CalledSymbol name of dynamic symbol we are calling (can be
  ///         extracted from llvm::Value with extractFromConstantStringPtr)
  /// \return a description of the call and boolean indicating whether the call
```

Reply:
I think it was clang-format's idea, but I'll change it.